src/common/NFTget.ts

```diff
@@ -276,11 +277,15 @@ export async function NFTGet(
   const validNFTs = filterOutIncompleteNFTs(nfts);
   let finalNFts = validNFTs;
 
+  if (filterNFTFunc){
```

Comment:
lets make this function name more descriptive (something like filterForOpenTickets)?

Reply:
hmm - I was trying to name it generically because the passed function to NFTGet() as a parameter could be any defined function -> filterForOpenTickets or filterForClosedTickets etc...

how about  filterForFunc, filterForMappedFunc - are these more descriptive to reflect this ^?

Reply:
Got it, sounds good!